### PR TITLE
[Docs] Add links for missing languages

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Comment by iHiD** [comment]
I'd argue that if we do this, all the tracks need this checking.
```
 [ ] Ensure there is a link to your track's GitHub issues on the [main README.md](../../README.md)
```

**Comment by ErikSchierboom** [comment]
That sounds like a good idea. Should I add them in this PR?

**Comment by ErikSchierboom** [comment]
> I'd argue that if we do this, all the tracks need this checking.

That, or I should be more careful when bootstrapping tracks...

File: languages/cpp/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Before we publicize requesting contribution for this language, the following ste
 
 - [ ] Have a kick-off discussion between track maintainers
 - [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
-- [ ] Ensure there is a link to your track's GitHub issues on the [main README.md](../../README.md))
+- [x] Ensure there is a link to your track's GitHub issues on the [main README.md](../../README.md))
```
Comment (iHiD) [inline languages/cpp/README.md]: Want to fix the `))` while you're here?

Comment (ErikSchierboom) [reply languages/cpp/README.md]: Yep. It was there on two places

**Comment by coriolinus** [comment]
I note that there's no `track/rust` label attached, though.

File: README.md [hunk 2]
```diff
@@ -17,18 +17,31 @@ We are busy working away on the product and development work to add all this fun
...
 _(This list is sorted alphabetically.)_
 
+- [Bash](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fbash)
 - [C](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fc)
 - [C#](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fcsharp)
 - [Clojure](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fclojure)
 - [Common Lisp](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fcommon-lisp)
+- [Dart](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Fdart)
 - [Elixir](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Felixir)
+- [Emacs LISP](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Felisp)
```
Comment (sshine) [inline README.md]:
```suggestion
- [Emacs Lisp](https://github.com/exercism/v3/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted+label%3Atrack%2Felisp)
```

File: languages/_sidebar.md [hunk 3]
```diff
@@ -4,16 +4,28 @@
   - [Bash](/languages/bash/README.md)
   - [C](/languages/c/README.md)
   - [C#](/languages/csharp/README.md)
+  - [Clojure](/languages/clojure/README.md)
+  - [Common Lisp](/languages/common-list/README.md)
   - [Dart](/languages/dart/README.md)
+  - [Elixir](/languages/elixir/README.md)
+  - [Emacs LISP](/languages/emacs-lisp/README.md)
```
Comment (sshine) [inline languages/_sidebar.md]:
```suggestion
  - [Emacs Lisp](/languages/emacs-lisp/README.md)
```

Comment (ErikSchierboom) [reply languages/_sidebar.md]: I'll fix this in a separate PR, as there are multiple places this has to be changed.

**Comment by ErikSchierboom** [comment]
> When you request a review from everyone, does that mean you want everyone to see the change, but for anyone to merge it? Or do you want for everyone to approve before you merge? Or do you expect some fixed amount of people to approve before you yourself merge?

Well, this happens automatically due to the auto-labeling. But I agree, we need to define a policy here.

File: languages/_sidebar.md (hunk 3 above)
Comment (sshine) [reply languages/_sidebar.md]: I noticed that [exercism/elisp's config.json](https://github.com/exercism/elisp/blob/master/config.json) uses "Emacs Lisp", which stylistically coincides with the v3 "Common Lisp" track, so this is probably the way.

Comment (ErikSchierboom) [reply languages/_sidebar.md]: See https://github.com/exercism/v3/pull/151/files

**Comment by sshine** [comment]
> this happens automatically due to the auto-labeling

Ah! So if you only touched a single language, those maintainers would probably expect to be included, but because you're doing cross-repo scaffolding, the notifications are just a side-effect. Good to know.

**Comment by ErikSchierboom** [comment]
Yep, that's precisely what is happening! I considered doing individual PR's, but decided against it in this case.

**Pull request merged**